cirq-superstaq/example-requirements.txt

```diff
@@ -1,3 +1,2 @@
 notebook~=6.4.12
-pulser~=0.7.0
-qiskit>=0.35.0, <0.37.0
```

Comment:
so we're no longer using pulser or was this package merged into qiskit-terra?

Reply:
no longer using it - we just missed these lines in https://github.com/SupertechLabs/cirq-superstaq/pull/439 & https://github.com/SupertechLabs/qiskit-superstaq/pull/315